### PR TITLE
Fix Bug in `getTokenFromResponse_` for APIs that Respond with 201 Created

### PR DESCRIPTION
**Pull request by stvkas**
I tried using this library to connecting Google Apps Script and the [Medium API](https://github.com/Medium/medium-api-docs), and `getTokenFromResponse_` throws an error despite getting a successful response. The function currently throws an error if the response code isn't 200, but successful access token response from Medium have a 201 status code. So I modified that check to throw an error if the status code is outside of the entire 2xx success range.

This PR also includes examples for connecting to [Strava](https://strava.github.io/api/) and Medium.

### CODE REVIEW

**Comment by stvkas** [comment]
I signed it!

File: dist/OAuth2.gs [hunk 1]
```diff
@@ -27,7 +27,7 @@
 
 /**
  * The supported formats for the returned OAuth2 token.
- * @type {Object.<string, string>
+ * @type {Object.<string, string>}
```
Comment (oshliaer) [inline dist/OAuth2.gs]: Finally!

**Comment by erickoledadevrel** [comment]
Thanks for the fix, and the samples!

**Comment by stvkas** [comment]
No problem. Happy to help out!

**Pull request merged**